src/components/K8sContainerViewContainer/K8sContainerViewContainer.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * ************************************
+ *
+ * @module PodViewContainer.js
+ * @description Component to get information about each container in the current pod, and pass that information to graph components render details of a single K8s pod
+ * 
+ * ************************************
+ */
+
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import InputLabel from '@mui/material/InputLabel';
+
+import * as containerPromql from '../../utils/container-promql-util';
+import * as podPromql from '../../utils/pod-promql-util';
+import K8sContainersOverview from '../K8sContainersOverview/K8sContainersOverview';
+import K8sContainerHeading from '../K8sContainerHeading/K8sContainerHeading';
+import * as actions from '../../actions/actions'
+
+const primaryColor = '#25274D';
+ 
+
+const K8sContainerViewContainer = (props) => {
+
+  //access podNames state from store, and set allContainerNamesList state
+  const { podInfo } = useSelector(state => state.pod);
+  const { nodeNames } = useSelector(state => state.node);
+  const [ allContainerNamesList, setAllContainerNamesList ] = useState([]);
+  // keep track of current pod
+  // set first pod in pod names list as default if defined
+  const [ currentPod, setCurrentPod ] = useState(podInfo[0].podName || 'no pod selected');
+
+  //**START HERE **/
+  const [ currentPodInfo, setCurrentPodInfo ] = useState(podInfo[0] || []);
+
+  const dispatch = useDispatch();
+  //get array of pod names from prometheus server and use the array to update state
+  //get all cluster's containers using fetch request and update state
+  useEffect(async () => {
+    const podInfoList = await podPromql.fetchPodInfoList(nodeNames[0]);
```

Comment:
Changed in this instance (line 47), would like to come back to changing the PodInfo variable name from the Redux store since it is used in multiple files.